library/setup.js

```diff
@@ -4,10 +4,12 @@ import VerticalSectionOverflow from './../src/components/VerticalSectionOverflow
 import VerticalItem from './../src/components/VerticalItem';
 import GlobalHeader from './../library/exampleComponents/GlobalHeader';
 import CodeEditor from './pages/components/CodeEditor';
+import Tab from '../src/components/Tab/';
 
 global.MenuItem = MenuItem;
```

Comment:
remove all components here, only keep those that are not part of the components like GlobalHeader and CodeEditor